frontend/pages/admin/UserManagementPage/components/UserForm/UserForm.tsx

```diff
@@ -218,25 +221,28 @@ class UserForm extends Component<ICreateUserFormProps, ICreateUserFormState> {
     const {
       formData: { global_role },
     } = this.state;
+    const { isBasicTier } = this.props;
     return (
       <>
-        <InfoBanner className={`${baseClass}__user-permissions-info`}>
-          <p>
-            Global users can only be members of the top level team and can
-            manage or observe all users, entities, and settings in Fleet.
-          </p>
-          <a
-            href="https://github.com/fleetdm/fleet/blob/master/docs/1-Using-Fleet/2-fleetctl-CLI.md#osquery-configuration-options"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Learn more about user permissions
-            <img src={OpenNewTabIcon} alt="open new tab" />
-          </a>
-        </InfoBanner>
+        {isBasicTier && (
+          <InfoBanner className={`${baseClass}__user-permissions-info`}>
+            <p>
+              Global users can only be members of the top level team and can
+              manage or observe all users, entities, and settings in Fleet.
+            </p>
+            <a
+              href="https://github.com/fleetdm/fleet/blob/master/docs/1-Using-Fleet/2-fleetctl-CLI.md#osquery-configuration-options"
+              target="_blank"
+              rel="noreferrer"
+            >
+              Learn more about user permissions
+              <img src={OpenNewTabIcon} alt="open new tab" />
+            </a>
+          </InfoBanner>
+        )}
         <p className={`${baseClass}__label`}>Role</p>
         <Dropdown
-          value={global_role || "observer"}
+          value={global_role || "Observer"}
```

Comment:
Let's keep default values at the top of the component. At line 95, "Observer" can be the default instead of `null`.

Reply:
Could we open a new issue for moving default values to state?  I don't have the time to change this and since all I did was change capitalization to match mocks, it might be good to comb through for other default states together across the whole app.

Reply:
Created new issue #1039.